modules/ROOT/pages/configure-client-management-azure.adoc

```diff
@@ -28,7 +28,7 @@ The *Add OIDC Azure client provider* page appears.
 +
 For an example of the `PATCH` payload, see the https://docs.microsoft.com/en-us/graph/api/application-update?view=graph-rest-1.0&tabs=http[Update application] documentation. For an example of the `DELETE` payload, see the https://docs.microsoft.com/en-us/graph/api/application-delete?view=graph-rest-1.0&tabs=http[Delete application] documentation.
 +
-Client secrets updates are not currently supported.
+
 . Click *Create*.
 
 == Microsoft Identity Platform Support
```

Comment:
Please rename 'Microsoft Identity Platform' to 'Microsoft Entra ID ( previously Azure AD) platform"

Reply:
This can be done later in https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001sCzzOYAS/view